report-utils/README.md

```diff
@@ -0,0 +1,87 @@
+## Report Utilities
+
+Gatling produces a detailed HTML report from the simulation.log file. 
+However, our Apples to Apples test reports include a summarized comparison of the data from two test runs.
+Producing this report currently requires manually copying the data from each HTML report into the corresponding table in a Google Sheets doc.
+
+These utility scripts have been created to assist in that process. They are a work-in-progress. Some manual steps are still required, but this gets us closer to a fully automated process.
+
+#### extract-csv.rb
+
+This script parses the JSON results file for a specified test run and creates a CSV file with the data used in our Apples to Apples test report.
+
+How to use it:
+
+In this example the result folder for our performance run will be `gatling-puppet-load-test/PERF_1524862584`. 
+`PERF_1524862584` contains the following folders:
+
+* `ip-10-227-0-132.amz-dev.puppet.net`
+* `ip-10-227-3-213.amz-dev.puppet.net`
+
+If you know which is your metrics node, start there; otherwise, it will be the one containing a `root` folder.
+In our example the metrics node is `ip-10-227-0-132.amz-dev.puppet.net`. You'll need to provide the script with the full path to the result folder, for example:
+
+```
+bill.claytor:~> cd /Users/bill.claytor/RubymineProjects/gatling-puppet-load-test/report-utils 
+bill.claytor:~/RubymineProjects/gatling-puppet-load-test/report-utils> ls
+README.md		compare-results.rb	extract-csv.rb
+bill.claytor:~/RubymineProjects/gatling-puppet-load-test/report-utils> ruby extract-csv.rb /Users/bill.claytor/RubymineProjects/gatling-puppet-load-test/PERF_1524862584/ip-10-227-0-132.amz-dev.puppet.net/root/gatling-puppet-load-test/simulation-runner/results/PerfTestLarge-1524848074511
+Creating PerfTestLarge-1524848074511.csv
+bill.claytor:~/RubymineProjects/gatling-puppet-load-test/report-utils> ls
+PerfTestLarge-1524848074511.csv	README.md			compare-results.rb		extract-csv.rb
+bill.claytor:~/RubymineProjects/gatling-puppet-load-test/report-utils> 
+
+```
+
+If the script succeeds, a CSV file with the name of the performance run will be created. 
+In this example the file is 'PerfTestLarge-1524848074511.csv'. 
+
+#### compare-results.rb
```

Comment:
We should see if we can use the bigquery stuff that @pinkypie added to store results and do comparisons rather than using the csv files. That way we can easily compare to a known baseline build instead of a csv we have locally.

Reply:
I definitely agree! This was just a quick attempt to provide a semi-automated way to produce the current report format until we get to that point.